boards/arm/beagle_bbai64_r5/doc/index.rst

```diff
@@ -0,0 +1,96 @@
+.. _beaglebone_ai64:
+
+BeagleBone AI-64
+################
+
+Overview
+********
+BeagleBone AI-64 is a computational platform powered by TI J721E Soc, which is
+targeted for automotive applications.
+
+.. figure:: assets/bbai_64.png
+		:align: center
+		:width: 500px
+		:alt: BeagleBoard.org BeagleBone AI-64
+
+Hardware
+********
+The BeagleBone AI-64 is powered by TI J721E Soc, which has three domains (Main,
+MCU, WKUP). This document gives overview of Zephyr running on Cortex R5 in the
+Main domain.
+
+L1 Memory System
+----------------
+* 16 KB instruction cache.
+* 16 KB data cache.
+* 64 KB TCM.
+
+Region Address Translation
+--------------------------
+The RAT module performs a region based address translation. It translates a
+32-bit input address into a 48-bit output address. Any input transaction that
+starts inside of a programmed region will have its address translated, if the
+region is enabled.
+
+VIM Interrupt Controller
+------------------------
+The VIM aggregates device interrupts and sends them to the R5F CPU(s). The VIM
+module supports 512 interrupt inputs per R5F core. Each interrupt can be either
+a level or a pulse (both active-high). The VIM has two interrupt outputs per core
+IRQ and FIQ.
+
+Supported Features
+******************
+The board configuration supports,
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardwares features are currently not supported.
+
+The default configuration can be found in the defconfig file:
+
+``boards/arm/beagle_bbai64_r5/beagle_bbai64_r5_defconfig``
+
+Flashing
+********
+The Zephyr image can be flashed to J721E Cortex R5 through remoteproc from
+Linux (Running on A72).
+
+To build hello world application for BeagleBone AI-64:
+
+``west build -p auto -b beagle_bbai64_r5 samples/hello_world``
```

Comment:
Do you write the firmware file to /lib/firmware? What filename does it get? Do you set the filename through remoteproc temporarily? Will the firmware restart after reboot?

Reply:
Do you write the firmware file to /lib/firmware?
Yes, I copy the image to lib/firmware

What filename does it get?
zephyr.elf

Do you set the filename through remoteproc temporarily?
Yes, the image name is passed to remoteproc through sysfs.

Will the firmware restart after reboot?
Firmware does not reboot, if Linux is rebooted. But not if power or boot button is pressed to reset.